app/assets/javascripts/models/ui/project_list.js.coffee

```diff
@@ -308,4 +342,19 @@ ns.ProjectList = do (ko
     hideRelatedUrls: ->
       $('#related-urls-modal').modal('hide')
 
+    toggleMoreDetails: (data) =>
+      if data.collection_id not in @moreDetailsActive()
+        @moreDetailsActive.push(data.collection_id)
+      else
+        @moreDetailsActive.splice(@moreDetailsActive().indexOf(data.collection_id), 1)
+
+    toggleViewDownloads: (data) =>
```

Comment:
I'd prefer that we use more descriptive variable names here, `data` doesn't tell me what I am being provided.